features/scv/api/src/main/java/org/opennms/features/scv/api/SecureCredentialsVault.java

```diff
@@ -25,6 +25,17 @@
 
 public interface SecureCredentialsVault {
 
+    public enum KeyStoreType {
+        JCEKS,
+        PKCS12;
+
+        public static KeyStoreType fromSystemProperty() {
+            return KeyStoreType.valueOf(System.getProperty("org.opennms.features.scv.keystore", "JCEKS").toUpperCase());
```

Comment:
Good to use a constant for this ``` private final static String SCV_KEYSTORE_PROPERTY = "org.opennms.features.scv.keystore"; ```
```suggestion
            return KeyStoreType.valueOf(System.getProperty(SCV_KEYSTORE_PROPERTY, "JCEKS").toUpperCase());
```